src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVNKubernetes CNI plugin.
+* The active Gateway now performs periodic health checks on the connections to remote clusters, updates the Gateway
+  connection status, and adds latency statistics.
+* Gateways now export the following connection metrics on TCP port 8080 which can be used with Prometheus. These are
+  currently only supported for the Libreswan cable driver:
+  * The count of bytes transmitted and received between Gateways.
+  * The number of connections between Gateways and their corresponding status.
+  * The timestamp of the last successful connection established between Gateways.
+  * The RTT latency between Gateways.
+* The Libreswan cable driver is now the default.
+* The strongSwan cable driver is deprecated and will be removed in a future release.
+* The Lighthouse DNS always returns the IP address of the local ClusterIP Service, if available, otherwise it
```

Comment:
```suggestion
* The Lighthouse DNS always returns the IP address of the local ClusterIP Service, if available and exported, otherwise it
```
If service is available locally but not exported, we don't consider it.

Reply:
I made it "..._local exported ClusterIP Service_...".